### DISCUSSION TITLE
How to get aboslute serial path of the USB adapter

### DISCUSSION THREAD

**Discussion by Franckybel** [Koenkk/zigbee2mqtt#24030] [General]
Hello,

I'm new using linux, I would like to set zigbe2mqtt using absolute serial port path but I do not know how to find it.

When I type "lsusb" I can see my device with ID 'cf1:0030' 

How to get the full path "/dev/serial/by-id/....." ?

Thanks in advance for your support

![image](https://github.com/user-attachments/assets/2ced4578-3190-4923-96e8-1f8fa1698647)

**Comment by francisp2** [comment]
```
ls /dev/serial/by-id
```

**Reply by Franckybel** [reply]
Thank you
